src/main/java/org/zone/region/flag/entity/player/move/message/greetings/GreetingsFlag.java

```diff
@@ -0,0 +1,99 @@
+package org.zone.region.flag.entity.player.move.message.greetings;
+
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.zone.region.flag.Flag;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplay;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplayTypes;
+
+/**
+ * Flag for greetings
+ */
+public class GreetingsFlag implements Flag {
+
+    private @NotNull MessageDisplay messageDisplay = MessageDisplayTypes.CHAT.createCopyOfDefault();
+    private @NotNull Component text;
+    @Deprecated(forRemoval = true)
+    private @Nullable Component legacyText;
+
+    @Deprecated(forRemoval = true)
+    public GreetingsFlag() {
+        this(null);
+    }
+    @Deprecated(forRemoval = true)
+    public GreetingsFlag(@Nullable Component greetingsMessage) {
+        this.legacyText = greetingsMessage;
```

Comment:
This will make the message and display type null which are both @NotNull

Reply:
bruh it's @Nullable in my IDE?

Reply:
Lines 16 and 17. I have just pulled this branch to my IDE and it shows them as ``@NotNull``. Have you pushed your latest changes?

Reply:
yeah